survey/survey_ontology.ttl

```diff
@@ -0,0 +1,32 @@
+@prefix survey: <http://semanticweb.org/metadatasurveylogy/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+<http://semanticweb.org/metadatasurveylogy> rdf:type owl:Ontology ;
+      <http://purl.org/dc/terms/contributor> "Alina Werner" ,
+            "Lisa Rossgoderer" ,
+            "Philipp Ulrich" ,
+            "Hauke Hinrichs" ,
+            "Maria Nakhla" ;
+      rdfs:label "Semiconductor Survey Metadata Ontology"@en ;
+      owl:versionInfo 0.5 .
+
+###  http://semanticweb.org/metadatasurveylogy/belongs_to
+survey:belongs_to rdf:type owl:ObjectProperty ;
+      rdfs:domain survey:SurveyResultDataOffering ;
+      rdfs:range survey:SurveyServiceOffering .
+
+###  http://semanticweb.org/metadatasurveylogy/SurveyResultDataOffering
+survey:SurveyResultDataOffering rdf:type owl:Class ; 
+      rdfs:subClassOf gx:DataResource ;
+      rdfs:label "Survey result data offering" ; 
+      rdfs:comment "The survey result refers to the data obtained from conducting the survey. It represents the collective responses gathered from all survey participants in relation to the specific topic of the survey." .
+
+###  http://semanticweb.org/metadatasurveylogy/SurveyServiceOffering
+survey:SurveyServiceOffering rdf:type owl:Class ; 
+      rdfs:subClassOf gx:ServiceOffering ;
```

Comment:
Interesting idea to derive from ServiceOffering. I will think about it. 
Please change it to SoftwareResource for now as described in https://github.com/GAIA-X4PLC-AAD/gaia-x-compliant-claims-example to have a link to ServiceAccessPoint (which holds the survey url)